src/Concerns/FormatsPrices.php

```diff
@@ -9,55 +9,39 @@ trait FormatsPrices
 {
     /**
      * The defined custom formatters.
-     *
-     * @var array
      */
-    static protected $formatters = [];
+    static protected array $formatters = [];
 
     /**
      * Formats the given monetary value into the application's currently preferred format.
-     *
-     * @param array $arguments
-     * @return string
      */
-    static public function format(...$arguments)
+    static public function format(...$arguments): ?string
     {
         return static::callFormatter(null, ...$arguments);
     }
 
     /**
      * Formats the given monetary value into the application's currently preferred format.
-     *
-     * @param null|string $name
-     * @param array $arguments
-     * @return string
      */
-    static protected function callFormatter($name, ...$arguments)
+    static protected function callFormatter(?string $name, ...$arguments): ?string
     {
         return static::getAssignedFormatter($name)->call($arguments);
     }
 
     /**
      * Formats the given monetary value using the package's default formatter.
      * This static method is hardcoded in order to prevent overwriting.
-     *
-     * @param string $value
-     * @param null|string $locale
-     * @return string
      */
-    static public function formatDefault($value, $locale = null)
+    static public function formatDefault(mixed $value, ?string $locale = null): string
     {
         return static::getDefaultFormatter()->call([$value, $locale]);
     }
 
     /**
      * Formats the given monetary value using the package's default formatter.
      * This static method is hardcoded in order to prevent overwriting.
-     *
-     * @param mixed $formatter
-     * @return \Whitecube\Price\CustomFormatter
      */
-    static public function formatUsing($formatter) : CustomFormatter
+    static public function formatUsing(mixed $formatter): CustomFormatter
```

Comment:
`$formatter` can be `string`, `callable` or `CustomFormatter`, which could be specified for IDE users instead of `mixed`.

```suggestion
    static public function formatUsing(string|callable|CustomFormatter $formatter): CustomFormatter
```